Dockerfile

```
@@ -1,6 +1,11 @@
+ARG TAG_VERSION=19.03.13
+ARG ALPINE_VERSION=3.13
+
+FROM docker:$TAG_VERSION as docker-base
+
 FROM alpine:3.13
```

Should this be

```suggestion
FROM alpine:$ALPINE_VERSION
```

?

---

pushed too soon! other errors too